src/python/pants/backend/project_info/dependees.py

```diff
@@ -35,8 +35,6 @@ async def map_addresses_to_dependees() -> AddressToDependees:
     address_to_dependees = defaultdict(set)
     for tgt, dependencies in zip(all_explicit_targets, dependencies_per_target):
         for dependency in dependencies:
-            # TODO(#10354): teach dependees how to work with generated subtargets.
```

Comment:
As discussed over DM, I think this experience is still a little finicky. In our repo, `./pants dependees src/python/pants/util:util` is empty. I would expect it to return all targets that use any of the files belonging to `:util`.

If I add an explicit dep on `util:util`, though, then that will show up.

I think this is likely fine, but unexpected.

Reply:
I view the changes to `./pants dependencies` and `./pants dependees` (ie, that both files and base targets are considered to be nodes in the graph) as a critical and intentional part of this change. I agree that it is unexpected, but I think that that is part of the reason why it was important to make this change before 2.0. Thanks for updating the description correspondingly.